1.4/Source/VanillaPsycastsExpanded/Paths/Archotechist/Ability_AggressiveHeatDump.cs

```diff
@@ -12,16 +12,21 @@
 
     public class Ability_AggressiveHeatDump : Ability
     {
+        public override float GetRadiusForPawn() =>
+            Mathf.Ceil(Mathf.Min(this.pawn.psychicEntropy.EntropyValue / 20f, 9f * base.GetRadiusForPawn()));
+
+        public override float GetPowerForPawn() => DamageDefOf.Flame.defaultDamage *
+                                                   this.pawn.psychicEntropy.EntropyValue * base.GetPowerForPawn();
+
         public override void Cast(params GlobalTargetInfo[] targets)
         {
             base.Cast(targets);
-            var damageMultiplier = this.pawn.psychicEntropy.EntropyValue / 100f;
-            var explosionRadius = Mathf.Min(Mathf.Ceil(this.pawn.psychicEntropy.EntropyValue / 20f), 9f);
+            var explosionRadius = this.GetRadiusForPawn();
```

Comment:
I'm using a var here for readability, I'm sure if I called the same function several times it would be optimized out anyway - if you feel it should change, that's something I'm fine doing.

Same goes for the similar line in heat detonation.

Reply:
No the var is the right way to do it.